packages/api/src/schema/api-schema.yaml

```
@@ -3145,6 +3171,72 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/error"
+  "/data/usage/query":
+    get:
+      summary: Query usage metrics
+      description: |
+        Allows querying any metric in its raw form. 
+        This can be called from the frontend with a CORS key.
+      parameters:
+        - name: from
+          in: query
+          description: |
+            Start millis timestamp for the query range (inclusive)
+          schema:
+            type: integer
+        - name: to
+          in: query
+          description: |
+            End millis timestamp for the query range (exclusive)
+          schema:
+            type: integer
+        - name: timeStep
+          in: query
+          description: |
+            The time step to aggregate viewership metrics by
+          schema:
+            type: string
+            enum:
+              - hour
+              - day
+        - name: creatorId
+          in: query
+          description: |
+            The creator ID to filter the query results
+          schema:
+            type: string
+      responses:
+        "200":
+          description: A Usage Metric object
+          content:
+            application/json:
+              schema:
+                type: object
+                description: |
+                  A simplified metric object about aggregate usage of an
+                  user.
+                properties:
+                  UserID:
+                    $ref: >-
+                      #/components/schemas/usage-metric/properties/UserID
+                  CreatorID:
+                    $ref: >-
+                      #/components/schemas/usage-metric/properties/CreatorID
+                  DeliveryUsageMins:
+                    $ref: >-
+                      #/components/schemas/usage-metric/properties/DeliveryUsageMins
+                  TotalUsageMins:
+                    $ref: >-
+                      #/components/schemas/usage-metric/properties/TotalUsageMins
+                  StorageUsageMins:
+                    $ref: >-
+                      #/components/schemas/usage-metric/properties/StorageUsageMins
```

Do we not return all the fields here? Can just reference the whole schema:
```suggestion
                $ref: #/components/schemas/usage-metric
```

You are missing the `required` field for example. It was only done like this on the `/data/views/query/total` API because there, in fact, we don't return all the fields.